doc/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,149 @@
+////
+This is the base template for Hazelcast integration module guides.
+
+You can clone this repository, work on it and create your guide, and then push to a new repository.
+////
+
+:github-address: https://github.com/hazelcast-guides/springboot-tomcat-session-replication
+:hazelcast: Hazelcast IMDG
+:framework: Spring Boot
+
+// Use this relative url if you are going to publish the guide on the guides site.
+// Note that this url will not work locally and raise asciidoctor errors.
+// So, complete the guide with the above url and set the below one just before 
+// publishing on the guides site.
+//
+:templates-url: templates:ROOT:page$
+
+= Spring Boot Tomcat Session Replication using Hazelcast
+
+// Content entered directly below the header but before the first section heading is called the preamble.
+
+include::{templates-url}/link-to-repo.adoc[]
+
+== What You’ll Learn
+
+Spring Boot is a framework that helps to build microservices easily. In a microservice architecture, there are different options to share data among the running services: one of them is caching, which Spring Boot leverages. It also provides session scope data access using built-in Tomcat HTTP sessions. But when running multiple microservices, it quickly becomes a hassle to replicate sessions and share their data across microservices.
```

Comment:
I wonder why did you trim the [original content](https://github.com/hazelcast-guides/springboot-tomcat-session-replication-on-kubernetes#spring-boot-tomcat-session-replication-on-kubernetes-using-hazelcast) here.

Reply:
I can put the original content back. I wasn't sure if that would be too long, I actually liked it and had troubles trimming it.

Reply:
Let's bring it back. We can divide into multiple categories afterwards.

Reply:
Alright I will do it